_plugins/meltano_json_generator.rb

```diff
@@ -58,6 +43,24 @@ def generate(site)
       top_level_index[plugin_type] = plugin_type_index
     end
     generate_file(site, "/meltano/api/v1/plugins", "index", JSON.generate(top_level_index))
+
+    # Write plugin files
```

Comment:
I moved this after the index endpoint is populated so I can use the `defaults` mapping to get the default variant.